go.mod

```diff
@@ -4,31 +4,69 @@ go 1.20
 
 require (
 	github.com/avast/retry-go v3.0.0+incompatible
+	github.com/cortexproject/cortex v1.2.1-0.20200803161316-7014ff11ed70
 	github.com/ethereum/go-ethereum v1.10.26
 	github.com/go-resty/resty/v2 v2.7.0
 	github.com/google/uuid v1.3.0
+	github.com/grafana/loki v1.6.1
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/lib/pq v1.10.7
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.14.0
-	github.com/prometheus/common v0.39.0
+	github.com/prometheus/common v0.37.0
```

Comment:
This one as well is getting downgraded

Reply:
That's okay, we need this version to be compatible for both `Loki` and `Prometheus`